bin/node/bench/src/tempdb.rs

```diff
@@ -20,7 +20,7 @@ use kvdb::{DBTransaction, KeyValueDB};
 use kvdb_rocksdb::{Database, DatabaseConfig};
 use std::{io, path::PathBuf, sync::Arc};
 
-#[derive(Debug, Clone, Copy, derive_more::Display)]
+#[derive(Clone, Copy, Debug)]
```

Comment:
Why?

Reply:
This should be reverted

Reply:
This `Display` derive is useless, it will use `Debug` derive implementation by default if we don't specify the `#[display(fmt = "...")] attribute for the enum variables.